lua/telescope/builtin/lsp.lua

```diff
@@ -246,6 +276,7 @@ local feature_map = {
   ["code_actions"]      = "code_action",
   ["document_symbols"]  = "document_symbol",
   ["references"]        = "find_references",
+  ["definitions"]        = "definition",
```

Comment:
```suggestion
  ["definitions"]        = "goto_definition",
```
I think the corresponding LSP capability is slightly off. You can check them, for instance, with
```lua
print(vim.inspect(vim.lsp.buf_get_clients(0)[1].resolved_capabilities))
```
in which one of the table keys is `goto_definition`

Currently, it should not be able to go to a definition as the feature_map check always returns `false`.

Reply:
Yup! :+1::+1: